spec/tasks/gen/model_spec.cr

```diff
@@ -0,0 +1,36 @@
+require "../../spec_helper"
+
+include CleanupHelper
+
+describe Gen::Model do
+  it "generates a model" do
+    with_cleanup do
+      io = IO::Memory.new
+      model_name = "ContactInfo"
+      ARGV.push(model_name)
+
+      Gen::Model.new.call(io)
+
+      File.read("./src/models/contact_info.cr").
+        should contain("class ContactInfo < BaseModel")
+      io.to_s.should contain("contact_info.cr")
+      io.to_s.should contain("./src/models")
```

Comment:
Could these expectations be combined to “./src/models/comtact_info.cr”?

Reply:
Yeah, I think we can combine the expectation if we change the success message to be something like:
"Generated ./src/models/contact_info.cr"

Reply:
Oh I thought it would work without changing the success messages since it container the whole path, but maybe I misread the success message?

Reply:
It currently contains the path and filename separately.  The message would be something like "Done generating contact_info.cr in ./src/models/"

Reply:
Oh I see! How about, "Generated ContactInfo in ./src/model/contact_info.cr"?

Reply:
Sounds like a plan. Getting the model name in there too will be a nice touch.